contentcuration/contentcuration/frontend/channelEdit/components/QuickEditModal/EditBooleanMapModal.vue

```diff
@@ -147,19 +161,16 @@
         }
       },
       async handleSave() {
-        this.validate();
-        if (this.error) {
-          return;
-        }
-
         await Promise.all(
-          this.nodes.map(node => {
+          this.nodes.map(async node => {
             const fieldValue = {};
+            const currentNode = this.getContentNode(node.id);
```

Comment:
Here we can use this logic instead:
```javascript
            const currentNode = this.getContentNode(node.id);
            if (this.hasMixedCategories) {
              // If we have mixed categories remain the old ones, and
              // just add new categories if there are any
              Object.assign(fieldValue, currentNode[this.field] || {});
            }

            Object.entries(this.selectedValues)
              .filter(([key, value]) => value.length === this.nodeIds.length)
              .forEach(([key]) => {
                fieldValue[key] = true;
              });

```

With this: we are making a copy of the old values of the node (the current implementation remove old values if it is not in the `selectedValues` map), and we are just adding, but never removing values that are selected (filtering those keys that are selected across all nodes).